helm-chart/values.yaml

```diff
@@ -111,10 +121,10 @@ mtls:
   # Valid values: off, permissive, strict
   mode: "permissive"
 
-  # The CA/signing key TTL in hours(h) or minutes(m).
+  # The CA/signing key TTL in hours(h). Min value 24h. Max value 999999h.
   caTTL: "720h"
 
-  # The TTL of certificates issued to workloads in hours(h) or minutes(m).
+  # The TTL of certificates issued to workloads in hours(h) or minutes(m).  Max value is 999999.
```

Comment:
nit: this should be consistent with the one above and have `999999h` or else remove `h` from above.

Reply:
It could also be minutes, which is why we left off the character.